cypress/fixtures/login.json

```diff
@@ -1,4 +1,4 @@
 {
-  "email": "rijusougata13@gmail.com",
-  "password": "12345678"
+  "email": "sougatariju13@gmail.com",
+  "password": "123456"
```

Comment:
if we change the firebase project, will this fail the tests? If so, this is not the correct approach then.

Reply:
no tests are dependent on firebase components  other than dashboard component.firebase project change will impact only on dashboard components tests. Thats why I left only dashboard tests blank to discuss further( mentioned on #171). All the other tests is based on frontend like components rendering or not ,edit form working or not , buttons working as expected or not, non logged in user can access the site or not like this.